presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/alluxio/TestingAlluxioMetastoreObjects.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.metastore.alluxio;
+
+import alluxio.grpc.table.Database;
+import alluxio.grpc.table.FieldSchema;
+import alluxio.grpc.table.Layout;
+import alluxio.grpc.table.LayoutSpec;
+import alluxio.grpc.table.Partition;
+import alluxio.grpc.table.PartitionSpec;
+import alluxio.grpc.table.Schema;
+import alluxio.grpc.table.TableInfo;
+import alluxio.grpc.table.layout.hive.HiveBucketProperty;
+import alluxio.grpc.table.layout.hive.PartitionInfo;
+import alluxio.grpc.table.layout.hive.SortingColumn;
+import alluxio.grpc.table.layout.hive.Storage;
+import alluxio.grpc.table.layout.hive.StorageFormat;
+import alluxio.shaded.client.com.google.protobuf.ByteString;
+
+public class TestingAlluxioMetastoreObjects
```

Comment:
Please see https://github.com/prestosql/presto/blob/master/README.md > "Enable the following inspections"